src/chat/UnreadNotice.js

```diff
@@ -29,71 +29,145 @@ const styles = StyleSheet.create({
   }
 });
 
-const POSITIONS = {
-  top: 'top',
-  bottom: 'bottom'
-};
-
 const showAnimationConfig = {
   toValue: 1,
   duration: 400,
-  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99)
+  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99),
+  useNativeDriver: true
 };
 
 const hideAnimationConfig = {
   toValue: 0,
   duration: 400,
-  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99)
+  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99),
+  useNativeDriver: true
+};
+
+// Duration after notice should hide
+const HIDE_DELAY = 4000;
+
+// Used to debounce repeated calls of show and hide animations
+const debounce = (fn, duration) => {
```

Comment:
There's a `debounce` function in `lodash` that we might be able to use.

Otherwise could you please move this to `utils` and import it from there? Shouldn't live in `UnreadNotice`